src/brogue/Recordings.c

```diff
@@ -706,7 +712,7 @@ void promptToAdvanceToLocation(short keystroke) {
     unsigned long destinationFrame;
     boolean enteredText;
 
-    if (!rogue.playbackPaused || unpause()) {
+    if (rogue.playbackOOS || !rogue.playbackPaused || unpause()) {
```

Comment:
Looking through another issue and I've confused myself about this, where is playbackOOS unset when we continue through the recording? Does it not need to be?

Reply:
This is so you can input turn numbers during OOS. Before, it went through unpause, which returns false because rogue.playbackMode is already true.

Reply:
I see, but what I mean is, after we go to another turn number from out of sync, doesn't rogue.playbackOOS need to be reset to false so that things work as normal? In testing everything works fine, I'm just wondering how, lol

Reply:
Oh, didn't think about that. Pretty sure we have to restart the recording no matter what from here and everything in rogue gets reset, so we should be fine lol

Reply:
Right yes, of course, initRecording resets it, called by advanceToLocation. That's fine then, thanks!